pkgs/by-name/th/thorium/package.nix

```
@@ -0,0 +1,40 @@
+{
+  stdenv,
+  fetchurl,
+  undmg,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: rec {
+  pname = "thorium";
+  version = "M126.0.6478.231";
```

The `version` attribute of a package *must* start with a digit.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

```suggestion
  version = "126.0.6478.231";
```